build/debian.sh

```diff
@@ -41,8 +41,8 @@ LDFLAGS='-fPIE -pie -Wl,-z,relro -Wl,-z,now -lcares' \
                 DEPS_PYTHON_CFLAGS="`/usr/bin/python2.7-config --cflags`" \
                 DEPS_PYTHON_LIBS="`/usr/bin/python2.7-config --ldflags`" \
                 PYCXX_SRCDIR=/usr/share/python2.7/CXX/Python2 \
-              --with-apps \
               --with-telepathy \
+              --with-apps \
```

Comment:
Ok. I'll reset it to old valu for now then.

On Thu, Aug 25, 2016 at 9:01 AM, Daniel Pocock <notifications@github.com>
wrote:

> In build/debian.sh
> <https://github.com/resiprocate/resiprocate/pull/67#discussion_r76205651>:
>
> >                --with-telepathy \
> > +              --with-apps \
>
> please don't include debian.sh in this pull request
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/resiprocate/resiprocate/pull/67/files/4c25e36cbaa0418017b559a7289c4b80f72caeda#r76205651>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIukotXTiRJ-MnLV5_Y1F-muKK8xhozrks5qjVnxgaJpZM4JsqYa>
> .
>



-- 


*Balram PariyarathTech Head | Excel 2016*
Computer Science and Engineering (Batch: 2013-17)
Govt. Model Engineering College, Kochi-21

[image: ☎]:  +91 8129130866
✉: balbbrl@gmail.com <p.balram96@gmail.com>